e2e/builder/build_test.go

```diff
@@ -40,7 +40,7 @@ func TestKitKnativeFullBuild(t *testing.T) {
 
 func doKitFullBuild(t *testing.T, name string, dependencies ...string) {
 	WithNewTestNamespace(t, func(ns string) {
-		Expect(Kamel("install", "-n", ns).Execute()).To(Succeed())
+		Expect(Kamel("install", "-n", ns, "--olm=false").Execute()).To(Succeed())
```

Comment:
When I executed the test locally on my openshift cluster, the test would always bring down the current OLM version of the operator rather than the local version. This would not be an issue on the kind clusters, of course. Therefore, I left it in as a certain method of testing the local version?

Reply:
Yes, testing local version is cumbersome, but picking a single test doesn't seem like a robust solution. I think an environment variable may be used in that case. Also downstream execution of the test suite may make the assumption that the installation mode is consistent across the entire test suite.

Reply:
Done